Code/Tools/ProjectManager/Source/NewProjectSettingsScreen.cpp

```diff
@@ -259,11 +341,37 @@ namespace O3DE::ProjectManager
         return projectTemplateDetails;
     }
 
+    void NewProjectSettingsScreen::StartTemplateDownload(const QString& templateName)
+    {
+        m_downloadController->AddObjectDownload(templateName, DownloadController::DownloadObjectType::Template);
+        auto foundButton = AZStd::ranges::find_if(
+            m_templateButtons,
+            [&templateName](const QAbstractButton* value)
+            {
+                return value->property(k_templateNameProperty) == templateName;
+            });
+
+        if (foundButton != m_templateButtons.end())
+        {
+            (*foundButton)->ShowDownloadProgress(true);
+        }
+    }
+
     void NewProjectSettingsScreen::UpdateTemplateDetails(const ProjectTemplateInfo& templateInfo)
     {
         m_templateDisplayName->setText(templateInfo.m_displayName);
         m_templateSummary->setText(templateInfo.m_summary);
         m_templateIncludedGems->Update(templateInfo.m_includedGems);
+        m_downloadTemplateButton->setVisible(templateInfo.m_isRemote);
+        m_downloadTemplateButton->disconnect();
+        connect(m_downloadTemplateButton, &QPushButton::clicked, this, [=]()
```

Comment:
you can probably just use `[&, templateInfo]` you would only pass in the reference for *this and a copy of templateInfo.